src/Symfony/Component/PropertyInfo/Tests/Fixtures/NullExtractor.php

```diff
@@ -76,6 +77,17 @@ public function getProperties($class, array $context = array())
         $this->assertIsString($class);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function isInitializable(string $class, string $property, array $context = array()): ?bool
+    {
+        $this->assertIsString($class);
```

Comment:
What's the point of this?

Reply:
I added this for consistency with other methods, I'm not sure of the origin intent...

Reply:
Right. So type hints are added now so these checks are redundant